eventing/samples/github-events/README.md

```diff
@@ -134,6 +104,36 @@ kubectl apply -f eventing/samples/github-events/auth.yaml
     kubectl apply -f eventing/samples/github-events/githubsecret.yaml
     ```
 
+1.  Use Docker to build the sample code into a container. To build and push with
+    Docker Hub, run the following commands, replacing `{username}` with your
+    Docker Hub username. Run these commands, r following from the _root_ of the
```

Comment:
Please revise this line and delete the typo/text: ", r following"

The resulting text should be:
"Run the following commands from the _root_ directory of the `knative/docs` repo:"